diff --git a/Makefile b/Makefile
index 2361028..4a3a049 100644
--- a/Makefile
+++ b/Makefile
@@ -22,6 +22,7 @@ LUA_INCLUDE_DIR ?=   $(PREFIX)/include
 LUA_CMODULE_DIR ?=   $(PREFIX)/lib/lua/$(LUA_VERSION)
 LUA_MODULE_DIR ?=    $(PREFIX)/share/lua/$(LUA_VERSION)
 LUA_BIN_DIR ?=       $(PREFIX)/bin
+LIBS =               -lluajit-5.1
 
 ##### Platform overrides #####
 ##
@@ -97,7 +98,7 @@ all: $(TARGET)
 doc: manual.html performance.html
 
 $(TARGET): $(OBJS)
-	$(CC) $(LDFLAGS) $(CJSON_LDFLAGS) -o $@ $(OBJS)
+	$(CC) $(LDFLAGS) $(CJSON_LDFLAGS) -o $@ $(OBJS) $(LIBS)
 
 install: $(TARGET)
 	mkdir -p $(DESTDIR)$(LUA_CMODULE_DIR)
